### ISSUE TITLE
Classes are incorrectly highlighted

### ISSUE THREAD

**Issue by fleischie** [styled-components/vim-styled-components#24]
Something I have not yet thought about: class definitions, that can be injected globally.

```javascript
import { css, injectGlobal } from 'styled-components'

const globalCss = css`
  .very-small-margin {
    margin: 1000vw;
  }
`;

injectGlobal`
  ${globalCss}
`;
```

I should highlight class names (and ids for that matter) independent from the actual name and only outside of the definition region. And anyway, why is that not already handled correctly?

**Comment by fleischie** [comment]
After 2 minutes of research, re-defining `cssClassName` and `cssIdentifier` fixes this problem, but they are the same as the original file, why isn't it used in the `@CSS` cluster?

**Comment by mike3run** [comment]
TIL that 💅 has global css thanks a lot!!

**Comment by fleischie** [comment]
See [comment on issue #21](https://github.com/styled-components/vim-styled-components/issues/21#issuecomment-330175614) for a discussion on completely re-writing this plugin to be more robust on these kind of things.

**Comment by fleischie** [comment]
Please see, whether #30 fixes this issue.

**Comment by fleischie** [comment]
Bump: Should be fixed, when #30 is merged. Anyways, please create a new issue, when merged. 😛
